src/background/Wallet/Wallet.ts

```diff
@@ -1203,7 +1204,10 @@ export class Wallet {
           context,
           params: { typedData, ...transactionContextParams },
         });
-        const rawTransaction = serializePaymasterTx({ transaction, signature });
+        const rawTransaction = serializePaymasterTx({
```

Comment:
Shouldn't we create `typedData` from transaction without `authorizationList` too 5 lines above?

Reply:
It shouldn't be there anyway, but good catch! It's more proper to do this above